src/main/java/xyz/nucleoid/plasmid/game/GameWorld.java

```diff
@@ -100,33 +133,63 @@ public void setGame(Game game) {
         this.invoker(GameOpenListener.EVENT).onOpen();
     }
 
+    /**
+     * Closes the current {@link Game} instance of this game world.
+     *
+     * <p>All {@link GameCloseListener#EVENT} listeners are notified of the close.
+     */
     private void closeGame() {
         this.invoker(GameCloseListener.EVENT).onClose();
-        this.listeners = new EventListeners();
-        this.rules = GameRuleSet.empty();
+        this.game = new Game();
     }
 
+    /**
+     * Attempts to add the given {@link ServerPlayerEntity} to this game world.
```

Comment:
```suggestion
     * Attempts to add the given {@link ServerPlayerEntity} to this game world if it is not already added.
```

Reply:
Thanks. Part of me thinks I should change it to just be `Adds the given {@link ServerPlayerEntity} to this game world if it/they is not already added/present.`, but I think your version works for now.